### PR TITLE
feature/environment

### PR DESCRIPTION
**Pull request by LeideVanessaC**

### CODE REVIEW

**Comment by DanielGuillen88** [comment]
removed #19

**Comment by GoteoMamez** [comment]
Need to change name to feature/environment

**Comment by JackDev21** [comment]
change title pull-request to feature/environment

File: staff/leide-vanessa-cardoso/gitignore
Comment (breadislife) [inline staff/leide-vanessa-cardoso/gitignore:1]: file needs to be named .gitignore

code needs to be corrected aswell
``` 
.DS_Store
thumbs.db
```

Comment (manuelbarzi) [reply staff/leide-vanessa-cardoso/gitignore:1]: delete this file

**Pull request merged**